docker/Dockerfile

```diff
@@ -73,6 +79,10 @@ RUN cargo pgrx package --features icu --pg-config "/usr/lib/postgresql/${PG_VERS
 
 FROM builder AS builder-pg_analytics
 
+# Intall the pgrx version of pg_analytics
+RUN cargo install --locked cargo-pgrx --version "${PGRX_VERSION_PG_ANALYTICS}" && \
+    cargo pgrx init "--pg${PG_VERSION_MAJOR}=/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/pg_config"
+
```

Comment:
```bash
# TODO: Once we start publishing releases for `paradedb/pg_analytics`, switch to a specific version tag
RUN git clone --branch main https://github.com/paradedb/pg_analytics.git /tmp/pg_analytics/
COPY shared/ /tmp/shared
```

This can be fixed now. We can remove the COPY shared/ and we can git clone --branch 0.1.1 for `pg_analytics`, which will make sure we have a snapshot of the code working even as we push new updates to `main`